test/aleph/http/server_test.clj

```diff
@@ -0,0 +1,17 @@
+(ns aleph.http.server-test
+  (:require [aleph.http.server :as server]
+            [clojure.string    :as str]
+            [clojure.test      :refer [deftest is]]))
+
+(deftest test-invalid-value-exception
+  (is (str/starts-with?
+       (ex-message (server/invalid-value-exception nil nil))
+       "Cannot treat nil of null as a response to '{}'.\nRing response expected."))
+  (is (str/starts-with?
+       (ex-message (server/invalid-value-exception {:uri "http://localhost/hello"
+                                                    :request-method :get} nil))
+       "Cannot treat nil of null as a response to '{:uri \"http://localhost/hello\", :request-method :get}'.\nRing response expected."))
```

Comment:
"nil of null" sounds weird. Can we skip showing the type when it's nil?

Reply:
https://github.com/clj-commons/aleph/pull/666/commits/53e3003c8a90809dad65ceb5b73815796f6d495b